tachyon/math/polynomials/univariate/univariate_polynomial.h

```diff
@@ -89,6 +89,42 @@ class UnivariatePolynomial final
     return coefficients_.Evaluate(point);
   }
 
+  template <typename ContainerTy>
+  constexpr UnivariatePolynomial& DivByVanishingPolyInPlace(
+      const ContainerTy& roots) {
+    using DenseCoeffs = UnivariateDenseCoefficients<Field, kMaxDegree>;
+    static_assert(std::is_same_v<Coefficients, DenseCoeffs>,
+                  "Only defined for UnivariateDensePolynomial");
+    for (const Field& root : roots) {
+      UnivariatePolynomial divisor =
+          UnivariatePolynomial(Coefficients({-root, Field::One()}));
+      DivInPlace(divisor);
```

Comment:
Is it more efficient to perform multiple divisions for each root, rather than creating a single divisor with the roots and dividing just once?

Reply:
I'm not sure which one is more efficient. We should benchmark it!

Reply:
@TomTaehoonKim also mentioned about this and I think it's more natural to create a vanishig polynomial and divide at once!